infra/core/security/key-vault-secrets.bicep

```diff
@@ -15,6 +15,7 @@ targetScope = 'resourceGroup'
 // ========================================================================
 
 @description('The form of each Key Vault Secret to store.')
+@secure()
```

Comment:
I'm surprised there's only one file changed. I expected 3 files changed at minimum due to the type being spread across this file, the application-post-config.bicep file, and another type file in the types folder.

Reply:
oh - that type isn't actually declared anywhere else. I can add it to the types folder if that's best practice